contents/tutorials/delayed-survey.md

```diff
@@ -0,0 +1,66 @@
+---
+title: How to show a survey after a delay
+date: 2023-11-01
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys']
+---
+
+Before asking users about your app, you want them to **actually use it**. Surveys that pop up right away hinder their initial experience and get less useful responses as a result.
+
+To help you do prevent this, you can set up a delay surveys and this tutorial shows you how. It goes over how to create a survey targeting a HTML class in PostHog and then implement a delay using that class and survey in your app.
```

Comment:
Hmm this paragraph is still confusing. Its not clear why we target a survey with an HTML class in PostHog. We'd need to explain a bit on how PostHog works.

Maybe something like: "PostHog lets you target your survey based on HTML class. So our approach will show the class after a delay, so that...."

Also Im not sure the intro is the right place for this either. Feels like itll go into too much detail too quickly.

Reply:
Rewrote this and moved the class mention later